docs/paper.md

```diff
@@ -115,7 +119,7 @@ Table: A **curated selection** of the metrics, tools and statistical tests curre
 |
 | **Probability**       	|Scores for evaluating forecasts that are expressed as predictive distributions, ensembles, and probabilities of binary events.                 	|Brier Score [@BRIER_1950], Continuous Ranked Probability Score (CRPS) for Cumulative Distribution Functions (CDFs) (including threshold-weighting, see @Gneiting:2011), CRPS for ensembles [@Gneiting_2007; @Ferro_2013], Receiver Operating Characteristic (ROC), Isotonic Regression (reliability diagrams) [@dimitriadis2021stable].              	  
 |
-| **Categorical**         |Scores for evaluating forecasts of categories.                 |Probability of Detection (POD), Probability of False Detection (POFD), False Alarm Ratio (FAR), Success Ratio, Accuracy, Peirce's Skill Score [@Peirce:1884], Critical Success Index (CSI), Gilbert Skill Score [@gilbert:1884], Heidke Skill Score, Odds Ratio, Odds Ratio Skill Score, F1 Score, Symmetric Extremal Dependence Index [@Ferro:2011], FIxed Risk Multicategorical (FIRM) Score [@Taggart:2022a].        	  
+| **Categorical**         |Scores, including contingency table metrics, for evaluating forecasts of categories.                 |Probability of Detection (POD), Probability of False Detection (POFD), False Alarm Ratio (FAR), Success Ratio, Accuracy, Peirce's Skill Score [@Peirce:1884], Critical Success Index (CSI), Gilbert Skill Score [@gilbert:1884], Heidke Skill Score, Odds Ratio, Odds Ratio Skill Score, F1 Score, Symmetric Extremal Dependence Index [@Ferro:2011], FIxed Risk Multicategorical (FIRM) Score [@Taggart:2022a].        	  
```

Comment:
After running this by Tennessee, I just added the reference to "including contingency table metrics". My logic was if people are searching for contingency table metrics, it would be good if they could easily find that `scores` contains contingency table metrics. I am open to alternative wording if someone has a better suggestion.

Reply:
I have checked how the JOSS PDF renders with the change to the table, and the table rendering is still fine. The change also hasn't impacted which content sits on which page of the PDF.

Reply:
I have modified this change to include brackets rather than commas, as in: "Scores (including contingency table metrics) for evaluating forecasts of categories.", as Tennessee and I think that looks better in the PDF. Again, if anyone has a suggestion for preferred wording/phrasing, please say so.

Reply:
Dealt with below, resolving this conversation.